R/as_noise_params.R

```diff
@@ -32,7 +32,7 @@ as_noise_params <- function(data, antigen_isos = NULL) {
     cli::cli_abort(
       class = "not noise_params",
       message = c(
-        "Can't convert {.envvar data} to noise_params.",
+        "Can't convert {.arg data} to noise_params.",
         "i" = "The column(s):{.strong {.var {missing_cols}}} are missing.",
```

Comment:
Please use `cli`'s [pluralization](https://cli.r-lib.org/reference/pluralization.html) features:

- [ ]  instead of `column(s)`, write `column{?s}`
- [ ] instead of `are missing`, write `{?is/are} missing`

Also: 
- [ ] put a space between the colon (`:`) and the opening bracket (`{`).